### PR TITLE
Extend pact helpers

### PR DESCRIPTION
**Pull request by amalkov**
Add shortcuts like_uuid, like_datetime, like_date

### CODE REVIEW

File: lib/pact/helpers.rb [hunk 1]
```diff
@@ -29,5 +29,17 @@ def each_like content, options = {}
       Pact::ArrayLike.new(content, options)
     end
 
+    def like_uuid uuid
+      Pact::Term.new(generate: uuid, matcher: /[0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}/)
+    end
+
+    def like_datetime datetime
+      Pact::Term.new(generate: datetime, matcher: /\d{4}\-\d{2}\-\d{2}T\d{2}:\d{2}:\d{2}([+-]\d{2}:\d{2})?/)
```
Comment (Antti) [inline lib/pact/helpers.rb]: Should it also validate if it's a valid date/time?
`2015-55-66T71:72:73` should not be considered as a valid date-time.

Comment (amalkov) [inline lib/pact/helpers.rb]: Agreed, updated Regex for date and datetime matchers, not sure that we need to be more strict hire. New Regex not ideal, but much better :)

**Comment by bethesque** [comment]
@uglyog how does this fit with the terminology you've used in pact JVM?

**Comment by uglyog** [comment]
I used GUID, but it should be UUID. The regex is slightly different, but functionality is the same.

For dates, times and timestamps I'm not using regular expressions, but JVM time/date parsers.

**Comment by amalkov** [comment]
Hi, @bethesque. Do you have any concern?

**Comment by bethesque** [comment]
Nope, just a lack of time! Thanks for submitting it.

**Comment by amalkov** [comment]
Thanks

**Pull request merged**